configuration/src/lib.rs

```diff
@@ -18,6 +20,11 @@ where
     let path_root = find_configs_root().await?;
     load_env(path_root.clone()).await?;
     let common_config = read_toml_file(path_root).await?;
+
+    if let Err(validation_errors) = common_config.validate() {
```

Comment:
I expect configuration validation to panic and prevent the service from running with a non-valid configuration.
It is better to see a panic and error message than to see a warning that may just get lost in a large number of logs.